packages/wallet/backend/src/auth/service.ts

```diff
@@ -54,21 +57,44 @@ export class AuthService implements IAuthService {
     return user
   }
 
+  public async resendVerifyEmail({
+    email
+  }: resendVerifyEmailArgs): Promise<void> {
+    const user = await this.userService.getByEmail(email)
+
+    // TODO: Prevent timing attacks
+    if (!user) {
+      throw new Unauthorized('Invalid account')
```

Comment:
I recommend that we don't send an error in this case and just return a success response
user should not be able to understand if an account exists or not to reduce security risks